src/uct/ib/base/ib_iface.c

```diff
@@ -1374,6 +1383,28 @@ void uct_ib_iface_set_reverse_sl(uct_ib_iface_t *ib_iface,
     ib_iface->config.reverse_sl = (uint8_t)ib_config->reverse_sl;
 }
 
+static void
+uct_ib_iface_set_priority_sls(uct_ib_iface_t *ib_iface,
+                              const uct_ib_iface_config_t *ib_config)
+{
+    int i;
+
+    memset(ib_iface->config.priority_sls, ib_iface->config.sl, UCT_IB_SL_NUM);
+
+    for (i = 0; i < ib_config->priority_sls.count; ++i) {
+        ucs_assert(ib_config->priority_sls.priority_sls[i] < UCT_IB_SL_NUM);
+        ib_iface->config.priority_sls[i] =
+                (uint8_t)ib_config->priority_sls.priority_sls[i];
+    }
+}
+
+void uct_ib_iface_set_configured_sls(uct_ib_iface_t *ib_iface,
```

Comment:
IMO this wrapper is redundant

Reply:
Why? I think we can avoid code duplication issues with that wrapper.

Reply:
ok